lib/systems/default.nix

```
@@ -421,6 +433,9 @@ let
           isNoStdTarget =
             any (t: hasInfix t final.rust.rustcTarget) ["-none" "nvptx" "switch" "-uefi"];
         };
+      } // {
+         _type = "elaborated-system";
```

I call these "platform objects", but that might just be me. They're definitely not system strings though. Having a `system` in a system object would be confusing to talk about, especially in actually vocal speak, in person.
```suggestion
         _type = "platform";
```



I don't think non-elaborated ones will ever be type-tagged, but if you insist, you may still consider

```suggestion
         _type = "elaborated-platform";
```

---

I was kinda unsure how to name it myself. It seemed "system" and "elaborated system" was used internally. But I agree, "platform" is clearer IMO.